stdlib/experimental/experimental.flux

```diff
@@ -28,14 +237,171 @@ builtin to : (
     ?token: string,
 ) => [A] where A: Record
 
-// An experimental version of join.
+// join joins two streams of tables on the **group key and `_time` column**.
```

Comment:
Did you mean to bold here?

Reply:
Yes, joining on the group key and time is they key to this functions performance improvements and is also the key differentiator between `experimental.join` and `join`. With `experimental.join`, the only way to control what data is joined on is by changing the group key. There is a trade-off between performance and flexibility.